### DISCUSSION TITLE
Add optional tab for displaying all contents in other tabs

### DISCUSSION THREAD

**Discussion by myhades** [kinghat/tabbed-card#123] [Feature Requests]
Hi, thanks for the great work!
Currently using it to divide the room by device types.
Would it be possible to add a feature where user can optinally uses a "global" tab to display all contents in other tabs in order?
I know you can just copy n paste cards but that would be bad for maintenance.
It would be great to enter a room view with all devices and then use tabs to navigate through devices, if neccessary.
Thanks again. ;)

**Comment by ildar170975** [comment]
No need to copy/paste cards.
Use yaml anchors for repeating code.

**Comment by myhades** [comment]
> No need to copy/paste cards. Use yaml anchors for repeating code.

Also with decluttering card. That's more of a workaround.
I think it should be a native feature of this card, instead of using yaml anchors or decluttering card for each tab, which adds considerable complexity on a large scale. And both can't be easily maintained in the UI.

**Comment by ildar170975** [comment]
So, you want to add a considerable complexity into the tabbed-card instead - just for one small use-case.
BTW, there are plenty of variants of layouting cards - stack, grid, so this functionality then should be implemented in the card too...
For this FR it is required to specify a desired layout then. It probably should be a user-defined option like `layout: vertical-stack` (or `horizontal-stack`, or `grid` - and also there should be a possibility to define grid settings in the latter case).

**Comment by kinghat** [comment]
im still not sure i understand the feature request :sweat_smile: im going to move it to the proper discussion area though.
